reporters_db/data/reporters.json

```diff
@@ -13627,6 +13627,25 @@
             "variations": {}
         }
     ],
+    "N.Y. Slip Op.": [
+        {
+            "cite_type": "state",
+            "editions": {
+                "N.Y. Slip Op.": {
+                    "end": null,
+                    "start": "1750-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:ny;court.appeals"
```

Comment:
This isn't your addition but, does the court of appeals actually issue slip opinions?

Reply:
Almost certainly not. I've never seen that. I haven't seen any intermediate appellate slip opinions either; I've only seen trial court opinions in NY as "slips".

Reply:
I actually think NY CoA and AD opinions do initially come out as slip ops, but they are quickly "reported", and so realistically, one isn't going to encounter them cited as slip ops. https://casetext.com/case/arons-v-jutkowitz-3 (you can see it has a slip op number)

So ultimately, a citation to a Slip Op indicates the opinion is a trial court opinion with a very high degree of confidence.